examples/yum_collection_migration_spec.yml

```diff
@@ -7,10 +7,6 @@ yum_collection:
     - yumdnf.py
   modules:
     - packaging/os/yum.py
-  units:
```

Comment:
FTR I'm wiping these entries in my schema PR: https://github.com/bcoca/gravity/pull/6/files#diff-9436d08f8a9a7146dda153bc5f0c8943R142